lib/overload.t

```diff
@@ -1,17 +1,18 @@
 #!./perl -T
 
+use Config;
 BEGIN {
     chdir 't' if -d 't';
     @INC = '../lib';
-    require Config;
-    if (($Config::Config{'extensions'} !~ m!\bList/Util\b!) ){
+    if (($Config{'extensions'} !~ m!\bList/Util\b!) ){
 	print "1..0 # Skip -- Perl configured without List::Util module\n";
 	exit 0;
     }
 }
 
-my $no_taint_support = exists($Config::Config{taint_support})
-                     && !$Config::Config{taint_support};
+my $NoTaintSupport =
```

Comment:
Why the change of naming style? All the rest of this file is snake case.

Reply:
For consistency with the `$NoTaintSupport` in other files